topics/proteomics/tutorials/maxquant-msstats-tmt/tutorial.md

```diff
@@ -63,34 +63,21 @@ The raw data is available via the PRIDE repository under the ID: [PXD014145](htt
 >
 >    {% snippet faqs/galaxy/datasets_rename.md %}
 >
-> 4. To run MaxQuant, import the raw data from [PRIDE](https://www.ebi.ac.uk/pride/archive/projects/PXD014145).
->
+> 4. To run MaxQuant, import the raw data from [PRIDE](https://www.ebi.ac.uk/pride/archive/projects/PXD014145) as a 'Collection'. Set 'File Type' as 'thermo.raw' and click 'Start' and then 'Build'. Name the collection 'raw_files' and create it via 'Create collection'.
```

Comment:
Hi Niko,
a snippet with detailed instructions to upload data as a collection directly (instead of creating the collection from history items) is currently missing, but would be helpful.
If you want to create it, you could use it here :)

Reply:
Would love that!

Reply:
[`faqs/galaxy/datasets_import_via_link.md`](https://github.com/galaxyproject/training-material/blob/main/faqs/galaxy/datasets_import_via_link.md) is exactly what we need here I guess?